firstLaba/KeyApproach.java

```diff
@@ -0,0 +1,70 @@
+package com.company;
+
+public class KeyApproach {
```

Comment:
what's the difference between this class and reverse? is there any way to make this code more universal?